pkg/handler/handler.go

```diff
@@ -290,7 +290,7 @@ func (h handler) lastoperation(w http.ResponseWriter, r *http.Request, params ma
 
 	// service_id is optional
 	if val, ok := queryparams["service_id"]; ok {
-		req.ServiceID = uuid.Parse(val[0])
+		req.ServiceID = val[0]
```

Comment:
Honestly, I don't think that we are even using it currently in the broker except as a log statement also we allow the operation field above to be an empty string. do you feel strongly about this one